Introduction to Coroutines and Channels/06_StructuredConcurrency.md

```diff
@@ -32,39 +32,39 @@ fun main() = runBlocking { /* this: CoroutineScope */
 }
 ```
 
-When we call `launch` inside `runBlocking`, we call it as an extension to the implicit receiver
+If we were to call `launch` inside `runBlocking`, we would call it as an extension to the implicit receiver
```

Comment:
I don't understand why 'if we were to call' here - it's clarification for the example right above when we call `launch` inside `runBlocking`.